src/mmrelay/meshtastic/connection.py

```
@@ -0,0 +1,1359 @@
+            facade.CONNECTION_RETRY_BACKOFF_BASE,
+        )
+    )
+    exponent = min(attempts, max_capped_attempt)
+    return min(
+        float(facade.CONNECTION_RETRY_BACKOFF_BASE**exponent),
+        float(facade.CONNECTION_RETRY_BACKOFF_MAX_SECS),
+    )
+
+
+def _get_connect_time_probe_settings(
+    active_config: dict[str, Any] | None, connection_type: str
```

Ruff (0.15.9)

[warning] Unused function argument: `connection_type`

(ARG001)